homeassistant/components/statistics/sensor.py

```diff
@@ -1,25 +1,26 @@
 """Support for statistics for sensor values."""
+from collections import deque
 import logging
 import statistics
-from collections import deque
 
 import voluptuous as vol
 
-import homeassistant.helpers.config_validation as cv
+from homeassistant.components.recorder.models import States
```

Comment:
We don't depend on the recorder integration in this integration. We probably should do that by updating manifest.json.

Reply:
The `recorder` is currently not listed as dependency in the `manifest.json` file:
https://github.com/home-assistant/home-assistant/blob/39bf1b5f5d1ecb710e7457edee78cdeb44327247/homeassistant/components/statistics/manifest.json#L6

Reply:
Right. I think we should add it.

Reply:
Sure thing.